config/locales/js-en.yml

```diff
@@ -82,6 +83,7 @@ en:
     label_collapse: "Collapse"
     label_collapsed: "collapsed"
     label_collapse_all: "Collapse all"
+    label_committed_at: "committed revision %{revision}"
```

Comment:
Is it acceptible to inject HTML through the interpolations? I've seen this happen sometimes in the backend, and while it's not pretty, I'd rather go down that road than to touch these labels soon again because the order just won't work in all languages for `<a href="foo">:label_committed_at</a> :label_at <a href="bar">{{ date }}</a>`.